libqpdf/QPDFWriter.cc

```diff
@@ -983,7 +983,7 @@ QPDFWriter::writeBinary(unsigned long long val, unsigned int bytes)
 void
 QPDFWriter::writeString(std::string const& str)
```

Comment:
Does it make sense for this to take `std::string_view` too?

Reply:
There's a case to be made for having part of qpdf 12 be to replace `std::string const&` with `std::string_view` in as many places as possible and to live with the slight performance penalty in the meantime to avoid lots of API clutter.

Reply:
> Does it make sense for this to take std::string_view too?

Agree. Updated

> There's a case to be made for having part of qpdf 12 be to replace std::string const& with std::string_view 

Agree